.circleci/cimodel/data/simple/ge_config_tests.py

```diff
@@ -0,0 +1,114 @@
+import cimodel.lib.miniutils as miniutils
+from cimodel.data.simple.util.versions import MultiPartVersion, CudaVersion
+
+
+DOCKER_IMAGE_BASIC = "/".join([
+    "308535385114.dkr.ecr.us-east-1.amazonaws.com",
+    "pytorch",
+    "pytorch-linux-xenial-py3.6-gcc5.4:9a3986fa-7ce7-4a36-a001-3c9bef9892e2",
+])
+
+DOCKER_IMAGE_GPU_MEDIUM = "/".join([
```

Comment:
NIT <- docker images for different CircleCI configs should be the same, shouldn't they?
```suggestion
DOCKER_IMAGE_CUDA_10.2 = "/".join([
```

Reply:
Some are different.  Not sure why.

Reply:
I think you've illustrated another benefit of code-generating these configs, though.  Now we can put all of the different docker image paths in one file to force the question: "why are we not using the same image?"  And there will be a single authoritative place to update the image path.